crates/biome_css_formatter/src/lib.rs

```diff
@@ -259,21 +263,49 @@ impl FormatLanguage for CssFormatLanguage {
 }
 
 /// Format implementation specific to JavaScript tokens.
```

Comment:
``
```suggestion
/// Format implementation specific to CSS tokens.
```

Reply:
lol I even changed the JSON one when I noticed this, but didn't update this one somehow.